extensions/ccui/base-classes/UIScale9Sprite.js

```diff
@@ -224,11 +224,11 @@ ccui.Scale9Sprite = cc.Scale9Sprite = cc.Node.extend(/** @lends ccui.Scale9Sprit
 
         if(file != undefined){
             if(file instanceof cc.SpriteFrame)
-                this.initWithSpriteFrame(file, rect);
+                this.initWithSpriteFrame(file, capInsets);
```

Comment:
Actually the second parameter is rect or capInset, when the first parameter is a sprite frame, rect is recognized as capInset. It's a little bit confusing, I will refactor the variable name